### PR TITLE
Component mobile - Feed cards & pill buttons

### PR DESCRIPTION
**Pull request by kitorer**

### CODE REVIEW

File: mobile/src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Pressable, Text, View } from 'react-native';
+
+function ButtonPill(props) {
```
Comment (rob-3) [inline mobile/src/components/Button.tsx:3]: I prefer `function ButtonPill({ name }) {` here and then just `name` below.

File: mobile/src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <Pressable className= "rounded-full bg-[#2D1C70] opacity-95 h-7 items-center justify-center" >
+        <Text className='text-white margin mx-5'>{props.name}</Text>
+    </Pressable>
+  );
+}
+
+function ButtonRounded(props) {
```
Comment (rob-3) [inline mobile/src/components/Button.tsx:11]: Same as above

**Review by rob-3 — CHANGES_REQUESTED** [review]
I see what you're doing with the slightly different classnames. I would just make one `Button` that does `...items-center justify-center ${props.className}` in a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) at the end.

File: mobile/src/components/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <Pressable className= "rounded-md bg-[#2D1C70] opacity-95 h-7 items-center justify-center" >
```
Comment (Rikimbili) [inline mobile/src/components/Button.tsx:13]: For the colors I think it is good practice to use predefined colors. Currently for web we are using slate and indigo colors included in tailwind (No need to extend the theme in the config file). 

This way you can easily pick the shade too (In this case it'd be something like bg-indigo-600)

File: mobile/src/components/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  let textStyle = 'text-white margin mx-5 font-semibold';
+
+  if (disabled) {
+    baseStyle += ' text-slate-50 bg-indigo-900/40 hover:bg-indigo-900/80';
+  }
+  //default style
+  else{
+    baseStyle += ' text-slate-50 bg-indigo-900 hover:bg-indigo-800 active:opacity-70';
+  }
+  pilled ? baseStyle += ' rounded-full' : baseStyle += ' rounded-md';
```
Comment (rob-3) [inline mobile/src/components/Button.tsx:26]: Any reason you're using one style here for disabled and another for pilled?

File: mobile/src/components/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { Pressable, Text, View } from 'react-native';
+/*
+@param name - name of the button
+@param addButtonStyle - add custom style to the button
+@param addTextStyle - add custom style to the text
+@param pilled - make the button pilled
+*/
```
Comment (rob-3) [inline mobile/src/components/Button.tsx:7]: This should start with `/**` to help editors pick it up as a docstring. You should also prepend each line with `*` and indent it so the `*`s line up.

```ts
/**
 * @param name - name of the button
 * @param addButtonStyle - add custom style to the button
 * @param addTextStyle - add custom style to the text
 * @param pilled - make the button pilled
 */
```

**Review by rob-3 — CHANGES_REQUESTED** [review]

**Review by rob-3 — CHANGES_REQUESTED** [review]
Looks good, just run prettier on this if you can and I'll be happy to approve

**Review by rob-3 — APPROVED** [review]

**Pull request merged**